lib/src/key/mod.rs

```diff
@@ -4,26 +4,28 @@
 /// crate::key::root::hb                 /!hb{ts}/{nd}
 /// crate::key::root::nd                 /!nd{nd}
 /// crate::key::root::ni                 /!ni
-/// crate::key::root::ns                 /!ns{ns}
+/// crate::key::root::ns                 /!ns{ns name}
```

Comment:
So name here is string, not u32 right? How have the prefixes changed? I am wondering if all keys are affected - I think the lq/nd ones aren't constructed from a shared prefix and instead are "static" declarations of namespace and database

Reply:
Yes this ns-name is str!

Many keys under the `ns::all` are affected, due to that the namespace "ID" is now u32.

I didn't touch lq/nd for now to reduce the scope of this PR. We might need to change lq and node in a similar manner if we do want to support quick renaming of these (but perhaps we don't need it? Do we really need to rename nodes and LQs?